cpp/cmake/thirdparty/get_jitify.cmake

```diff
@@ -18,8 +18,8 @@
 function(find_and_configure_jitify)
   rapids_cpm_find(
     jitify 2.0.0
-    GIT_REPOSITORY https://github.com/rapidsai/jitify.git
-    GIT_TAG cudf_0.19
+    GIT_REPOSITORY https://github.com/NVIDIA/jitify.git
+    GIT_TAG jitify2
```

Comment:
Should we ask the Jitify team to make a tag so that we're not pinned to a "floating" branch?

Reply:
Yes, I think that'd be great